### ISSUE TITLE
ModuleNotFoundError: No module named 'mne.externals'

### ISSUE THREAD

**Issue by guishuyunye-lyw** [autoreject/autoreject#279]
When I try to run 'from autoreject import AutoReject', I got this issue.

**Comment by sappelhoff** [comment]
Can you try `pip install -U autoreject` and see whether the error is still present afterwards? 

> MNE latest verstion

please run `mne sys_info` in your terminal and post the output formatted as a code block here, thanks!

**Comment by guishuyunye-lyw** [comment]
I tried and got same result.

![image](https://user-images.githubusercontent.com/20347818/178465684-0233f643-793b-49a1-a2e6-1c678839bebe.png)

**Comment by sappelhoff** [comment]
Please post the entire output of `mne sys_info` as a code block, not parts of it as a screenshot, that may help with debugging.

**Comment by guishuyunye-lyw** [comment]
Platform:         Windows-10-10.0.19043-SP0
Python:           3.8.10 (default, May 19 2021, 13:12:57) [MSC v.1916 64 bit (AMD64)]
Executable:       D:\ProgramData\Anaconda3\envs\RSA\python.exe
CPU:              Intel64 Family 6 Model 158 Stepping 10, GenuineIntel: 6 cores
Memory:           23.9 GB

mne:              1.0.3
numpy:            1.20.3 {blas=mkl_rt, lapack=mkl_rt}
scipy:            1.6.2
matplotlib:       3.5.1 {backend=module://matplotlib_inline.backend_inline}

sklearn:          0.24.2
numba:            0.53.1
nibabel:          3.2.1
nilearn:          0.8.0
dipy:             1.4.1
cupy:             Not found
pandas:           1.3.0
pyvista:          0.31.3 {OpenGL 4.5.0 NVIDIA 457.20 via GeForce GTX 1060/PCIe/SSE2}
pyvistaqt:        Not found
ipyvtklink:       0.2.1
vtk:              9.0.3
PyQt5:            5.9.2
ipympl:           Not found
pooch:            v1.6.0

mne_bids:         Not found
mne_nirs:         Not found
mne_features:     Not found
mne_qt_browser:   Not found
mne_connectivity: Not found

**Comment by sappelhoff** [comment]
Thanks. I don't see where the problem is: We are regularly running our test suite with the same software versions that you have specified, and all tests pass without us running into the import problem. I suspect you have some installation issue like anaconda looking for a package in an unexpected spot.

If I were you, I'd completely uninstall anaconda and re-install my work environment step by step. But perhaps somebody has a better idea. Good luck.

**Comment by guishuyunye-lyw** [comment]
You are right! After reinstalling the anaconda and related env, the error disappeared. Thank you so much!
